src/common/_utils/index.ts

```diff
@@ -56,12 +58,27 @@ const createParsedColorTheme = backgroundColor => ({
  * with appropriate chroma colors.
  * @param backGroundColor A plain CSS color string
  */
-export const parseCustomColor = (backgroundColor: string) => {
-  if (!chroma.valid(backgroundColor)) {
-    throw new Error("Provided color theme contatins invalid color values")
+export const parseCustomColor = (
+  customColor: CustomColor | CustomColorObject
+) => {
+  if (
+    (typeof customColor === "string" && !chroma.valid(customColor)) ||
+    (typeof customColor === "object" &&
+      customColor.color &&
```

Comment:
If they are `undefined`, `chrome.valid` will return `false`, so we do not need to check that explicitly.